components/page-header/check-props.js

```diff
@@ -53,13 +59,58 @@ if (process.env.NODE_ENV !== 'production') {
 				)}`
 			);
 		}
+
+		deprecatedProperty(
+			COMPONENT,
+			props.contentRight,
+			'contentRight',
+			'onRenderActions',
+			createDocUrl('onRenderActions')
+		);
+
 		deprecatedProperty(
 			COMPONENT,
 			props.navRight,
 			'navRight',
 			'onRenderControls',
 			createDocUrl('onRenderControls')
 		);
+
+		if (props.onRenderActions) {
+			renderFunctionReturnContentsLackDisplayName(
```

Comment:
Great way to add user affordance and go the extra mile for our consumers @kevinparkerson !